index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+    <header class="toolbar-container">
+        <div class="logo-container">
+          <a class="logo" href="">My Logo</a>
+        </div>
+        <nav class="nav_item">
+          <ion-icon class="menu" name="menu-sharp"></ion-icon>
+        </nav>
+      </header>
+  
+      <section class="content">
+        <h1 class="name">
+          I'm Mozamel.<br />Glad to see<br />
+          you!
+        </h1>
+        <p class="paragraph">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+  
+        <span class="connect">LETS CONNECT</span>
+  
+        <div class="social_media">
+          <ul class="media_icons">
+            <li>
+              <a href="#">
+                <img src="assets/Icons/twitter.svg" alt="Twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/Icons/linkedin.png" alt="LinkedIn" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/Icons/media.svg" alt="Media" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/Icons/cat.svg" alt="Github" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/icons/victory.svg" alt="Victory" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js">
+    </script>
+    </div>
+  </body>
```

Comment:
- [ ] Kindly re-inset the `script` tags right before the closing `body` tag, those `script` tags shouldn't be inside your code or your content of the page, insert them after the `div`.
```suggestion
     </div>
    <script
      type="module"
      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
    ></script>
    <script
      nomodule
      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js">
    </script>
  </body>
```